packages/react-image/src/components/Image/useImage.ts

```diff
@@ -1,18 +1,9 @@
 import * as React from 'react';
-/* eslint-disable @typescript-eslint/ban-ts-comment */
-// @ts-ignore Typings require esModuleInterop
-import { objectFitImages } from 'object-fit-images';
```

Comment:
How is this polyfill broken? In theory we still have not killed IE11 support, so we should probably fix the polyfill rather ?

Reply:
Styles are not applied, check `useImageStyles.ts`. We can fix it later based on ask.